controllers/topics.controllers.js

```diff
@@ -0,0 +1,10 @@
+const {fetchAllTopics} = require('../models/topics.models')
+
+
+function getAllTopics(req,res,next){
+    fetchAllTopics().then((body)=>{
+      res.status(200).send(body.rows)
```

Comment:
We should send back data to the client on a meaningful key e.g, they should be able to access it on body.topics :)